litmus-portal/frontend/src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import TextField from '@material-ui/core/TextField';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import useStyles from './styles';
+
+// This is the Password Input Component
+interface State {
+  password: string;
+  showPassword: boolean;
+}
+
+interface passwordProps {
+  textType: string;
+  value: string;
+  handleChange: (event: React.ChangeEvent<{ value: string }>) => void;
+}
+
+const PasswordInput: React.FC<passwordProps> = ({
+  textType,
+  value,
+  handleChange,
+}) => {
+
+  const [showPassword, setShowPassword] = React.useState(false);
+
+  const handleClickShowPassword = () => {
+    setShowPassword(true);
+  };
+
+  const handleMouseDownPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+    setShowPassword(false);
+  };
+  const classes = useStyles();
+  
+  return (
+      <div className={classes.inputArea}>
+        <TextField
+          id="standard-adornment-password"
```

Comment:
The "standard-adornment-password" isn't being used in the styles or referenced anywhere in this component, that can be removed